Simplify events table schema

c-w opened the pull request:
See the use case described in [c65e3b6](https://github.com/CatalystCode/project-fortis-services/commit/c65e3b6970bb4c767394f3e3e5179cfa607bfb1b):

By adding non-queryable columns to the events table that contain all of an events places and topics, we can re-build a full event just from the data that we get back from that table. This makes a number of queries easier and more performant, including the MessagesSchema.event query.

erikschlegel reviewed (commented):
I considered this... Don't you already have the `topics` and `placedids` from the tags tables?

c-w commented:
@erikschlegel Yes we do, but then we'd have to join the tables (two trips to Cassandra) every time we fetch an event by id which increases code complexity.

Most events will have few tagged topics and places so the added storage for these two columns should be minimal. Performance impact on Cassandra should also be minimal since we're not adding the columns to a key or indexing them.

What do you think?

c-w committed:
Simplify events table schema

See the use case described here:
https://github.com/CatalystCode/project-fortis-services/commit/c65e3b6970bb4c767394f3e3e5179cfa607bfb1b

erikschlegel commented:
Makes sense. LGTM
